src/lib/cache/cache.go

```diff
@@ -137,3 +138,35 @@ func Initialize(typ, addr string) error {
 func Default() Cache {
 	return cache
 }
+
+var (
+	// cacheLayer is the global cache layer cache instance.
+	cacheLayer Cache
+	// cacheLayerOnce is the once condition for initializing instance.
+	cacheLayerOnce sync.Once
+)
+
+// CacheLayer is the global cache instance for cache layer.
+//
+//nolint:revive // ignore the function name check
+func CacheLayer() Cache {
+	// parse the redis url for cache layer, use the default cache if not specify
+	redisCacheURL := os.Getenv("_REDIS_URL_CACHE")
+	if redisCacheURL == "" {
+		return cache
+	}
+
+	u, err := url.Parse(redisCacheURL)
+	if err != nil {
+		log.Fatal("failed to parse the redis url for cache layer, bad _REDIS_URL_CACHE")
+	}
+
+	cacheLayerOnce.Do(func() {
+		cacheLayer, err = New(u.Scheme, Address(redisCacheURL), Prefix("cache:"))
```

Comment:
use Initialize()?

Reply:
The `Initialize` will override the system default cache instance, here we only want to initialize cache layer instance.